ocsci/pytest_customization/tests/test_pytest.py

```diff
@@ -18,6 +18,9 @@ def test_help_message(testdir):
     Check that ``py.test --help`` output lists custom options of
     ocscilib pytest plugin.
     """
+    testdir.makeconftest(textwrap.dedent("""
+        pytest_plugins = ['ocsci.pytest_customization.ocscilib']
+    """))
```

Comment:
My initial feedback here is that this should not be necessary, as the pytest is executed within the same process, so this should not be necessary. Has it any effect?

Reply:
Executing unit tests inside of `run-ci` has side-effects that are not always desirable for unit tests; I think unit tests should be run using `py.test` directly - and with this PR, that does work - though @vasukulkarni asked that I not change `tox.ini` just yet.

Reply:
This doesn't answer my question.